src/classlibnative/bcltype/grisu3.cpp

```diff
@@ -0,0 +1,376 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+// File: grisu3.cpp
+//
+
+//
+
+#include "grisu3.h"
+#include <check.h>
+#include <math.h>
+
+// 1/lg(10)
+const double Grisu3::D_1_LOG2_10 = 0.30102999566398114;
```

Comment:
This is incorrect. `1 / log2(10)` is `0.301029995663981195...`, which (when rounded) is exactly representable in binary64 as `0.30102999566398120`